articles/integrations/signals-integration-troubleshooting.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Signals Integration Troubleshooting"
+description: "How to troubleshoot signals integration validation and batch errors."
+---
+
+Your Signals integegration processes your sending data in batches of events. Each batch is validated to make sure it work with our analytics. You can see the results for each batch from the [Signals Integration page](https://app.sparkpost.com/signals/integration). This article outlines the possible issues the integration could have. Please reach out to your Technical Account Manager for support in solving issue with your integration.
```

Comment:
> You can see the results for each batch
Do you think someone could mistake this for viewing the analytics results? Maybe "You can view whether or not ingested batches were successfully processed from the <page>" or something like that.